src/emulator/databaseEmulator.ts

```diff
@@ -86,7 +98,7 @@ export class DatabaseEmulator implements EmulatorInstance {
     const { host, port } = this.getInfo();
     try {
       await api.request("PUT", `/.settings/rules.json?ns=${instance}`, {
-        origin: `http://${host}:${[port]}`,
+        origin: `http://${host}:${port}`,
```

Comment:
I have no idea how this ever worked ...

Reply:
Probably because it would join the array with commas, which would just be the port number